client/src/Assets/styles/style.css

```diff
@@ -288,4 +292,112 @@ footer time {
 
 .tabs > li {
     margin:  0 1em;
+}
+
+
+/* Popup style */
+.popup-box {
+    position: fixed;
+    background: #00000050;
+    width: 100%;
+    height: 100vh;
+    top: 0;
+    left: 0;
+  }
+  
+  .box {
+    position: relative;
+    width: 70%;
+    margin: 0 auto;
+    height: auto;
+    max-height: 70vh;
+    margin-top: calc(100vh - 85vh - 20px);
+    background: #fff;
+    border-radius: 4px;
+    padding: 20px;
+    border: 1px solid #999;
+    overflow: auto;
+    box-shadow: 0.2em 0.4em 3px 1.5px rgba(147, 197, 252, 0.3);
+  }
+  
+  .close-icon {
+    content: 'x';
+    cursor: pointer;
+    position: fixed;
+    right: calc(15% - 30px);
+    top: calc(100vh - 85vh - 33px);
+    background: #ededed;
+    width: 25px;
+    height: 25px;
+    border-radius: 50%;
+    line-height: 20px;
+    text-align: center;
+    border: 1px solid #999;
+    font-size: 20px;
+  }
+
+/* Form */
+
+.form > div {
+    margin: 1em 0;
+}
+
+.form input[type="text"] {
+    width: 100%;
+    width: 100%;
+    border-radius: 1em;
+    border-color: var(--olor-light);
+    height: 2em;
+    outline: none;
+    border: none;
+    box-shadow: 0.2em 0.4em 3px 1.5px rgba(147, 197, 252, 0.3);
+}
+
+/* Toggle */
+.togglle-btn {
+    position: relative;
+}
+
+#toggle {
+    /* visibility: hidden; */
+    position: absolute;
+    top: -9999px;
+    left: -9999px;
+}
+
+#toggle:checked + #toggle-label {
+    background-color: #E0C3FC;
+}
+
+#toggle:checked + #toggle-label:before {
+    /* transform: translateX(90px); */
+    background-color: #FFF;
+    left: 40px;
+} 
+
+#toggle-label {
+    display: flex;
+    width: 60px;
+    height: 20px;
+    border: 2px solid #E0C3FC;
+    border-radius: 20em;
+    position: relative;
+    background-color: #fff;
+    transition: background-color .75s ease-in-out;
+    cursor: pointer;
+}
+
+#toggle-label::before {
+    transition: 0.2s;
```

Comment:
Would it be possible to set a transition property here and use shorthand to combine the transition-delay also?

Reply:
I think so